ui/pages/keyring-snaps/snap-account-detail-page/header.tsx

```diff
@@ -86,7 +87,7 @@ export const SnapDetailHeader = ({
                   color: TextColor.textAlternative,
                 }}
                 className=""
-                height={2}
+                height={BlockSize.TwoTwelfths}
```

Comment:
This seems like a mistake and it would get ridden over anyway? I think it could be removed

<img width="1132" alt="Screenshot 2023-07-21 at 10 45 29 AM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/b80d2fae-f0cc-4b50-8e99-f07485d2796b">

Reply:
Do we need to remove `height: auto` from the component?

Reply:
I think we remove `height={BlockSize.TwoTwelfths}` here to get this PR in. We can review the styles in a different issue